floodviz/posterior_build/hydro_thumbnail.js

```diff
@@ -0,0 +1,248 @@
+
+'use strict';
+/**
+ * @param {Object} options - holds options for the configuration of the hydrograph
+ *    Non-optional Keys include:
+ *        @prop 'height' v(int) - height of the graph
+ *        @prop 'width' v(int) - width of the graph
+ *        @prop 'data' v(list) - A list of objects representing data points
+ *        @prop 'div_id' v(string) - id for the container for this graph
+ *
+ * hydromodule is a module for creating hydrographs using d3. Pass it a javascript object
+ * specifying config options for the graph. Call init() to create the graph. Linked
+ * interaction functions for other figures should be passed to init in and object.
+ *
+ */
+var hydromodule = function (options) {
```

Comment:
Actually, it looks like they are already merged. A separate ticket for this would still be helpful IMO. There may be a decent amount of refactoring. Something to talk about at sprint planning I guess.